Ports/fribidi/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='fribidi'
+version='1.0.13'
+useconfigure='true'
+use_fresh_config_sub='true'
+archive_hash='7fa16c80c81bd622f7b198d31356da139cc318a63fc7761217af4130903f54a2'
+use_fresh_config_sub='true'
+workdir="fribidi-${version}"
+configopts=(
+    "--prefix=${SERENITY_INSTALL_ROOT}/usr/local"
```

Comment:
--prefix isn't passed to any other meson ports. is it strictly required?

Reply:
The local version worked, but apparently the PR I shipped is slightly problematic in various ways. I will take a look against what other meson ports did and try to clean things up to improve the port's quality.